### DISCUSSION TITLE
Cannot use Integrated data in SCENIC

### DISCUSSION THREAD

**Discussion by Lumskie** [aertslab/SCENIC#243] [R]
Hi!!
I am trying to use Seurat Integrated object in SCENIC to infer GRN. However as integration gives 2000 anchors in the data that is only getting added in the exprMat which is affecting the data. How do I work with integrated seurat object in SCENIC?

Thanks in advance!!

**Comment by s-aibar** [comment]
Sorry @Lumskie , I think I don't understand the question... 
Could you give us an example of what kind of input are you trying to provide? (e.g. maybe from Seurat tutorial?)
